testsuite/mockserver.py

```diff
@@ -1,16 +1,23 @@
 """Module for Mockserver integration"""
+import enum
 from urllib.parse import urljoin
 
 import httpx
 
 
+class ContentType(enum.Enum):
```

Comment:
We could possibly move this to utils and use this in more functions, but this does not need to be done in this PR